src/providers/airwatch-config/airwatch-config.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+import { fromPromise } from 'rxjs/observable/fromPromise';
+
+import { AirwatchConfigModel } from './airwatch-config.model';
+import { AppPreferences } from '@ionic-native/app-preferences';
+
+@Injectable()
+export class AirwatchConfigProvider {
+
+  constructor(private appPreferences: AppPreferences) {}
+
+  public getAirwatchConfig = (): Observable<AirwatchConfigModel> =>  {
+    return fromPromise(this.getConfig());
+  }
+
+  private getConfig = async(): Promise<AirwatchConfigModel> => {
+    return {
+      configUrl: await this.appPreferences.fetch('configUrl'),
```

Comment:
Assuming there's no fetch all?

Reply:
No there's a defaults option which will get all the default values but that restricts how we can use the config xml.